pkgs/by-name/lm/lmstudio/darwin.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchurl
+, undmg
+, lib
+, meta
+, pname
+, version
+}:
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+      url = "https://releases.lmstudio.ai/mac/arm64/${version}/b/latest/LM-Studio-${version}-arm64.dmg";
+      sha256 = "sha256-kb1XoTZjhCL1+CsV+r3/EN0srzIJ43H2VMZ779dVq1k=";
+    };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications
```

Comment:
Remove `mkdir`, use `install`.

Reply:
Thanks for your input. 
`install` seems not to copy directories (`LM Studio.app` is actually just a directory, not a file). 
Therefore, I would have to use `install` in [a very awkward way](https://unix.stackexchange.com/a/165471), I fear. Is this actually, how I should do that? 

I looked at [the spotify packaging](https://github.com/NixOS/nixpkgs/blob/ad63eec6c19c8686527f55692802c400165c7b7e/pkgs/applications/audio/spotify/darwin.nix) for reference.

Reply:
Fair enough!